src/Disks/DiskLocal.cpp

```diff
@@ -363,7 +380,7 @@ void DiskLocal::removeFile(const String & path)
 {
     auto fs_path = fs::path(disk_path) / path;
     if (0 != unlink(fs_path.c_str()))
-        ErrnoException::throwFromPath(ErrorCodes::CANNOT_UNLINK, fs_path, "Cannot unlink file {}", fs_path);
+        fs::remove(fs_path);
```

Comment:
why? it was already attempted to be removed in unlink

Reply:
If the file doesn't exist, the unlink will return an error.
I will revert this change. Because I use `removeFileIfExists` in the caller instead of `removeFile`